### DISCUSSION TITLE
Production and industrial facilities - problems with 47,01 IED/LCP emission dataset transformation

### DISCUSSION THREAD

**Discussion by AurelijaStu** [INSPIRE-MIF/helpdesk#126] [Q&A]
Hello,

I would like to ask for advice about the target schema which is appropriate for data transformation of the 47.01 IED / LCP emission dataset.
According to the Production and industrial facilities specification, the data should be transformed according to the ProductionAndIndustrialFacilities GML Application Schema (version: 4) and/or the ProductionAnIndustrialFacilitiesExtension Model GML Application Schema (version: 3).
I used the https://inspire.ec.europa.eu/schemas/pf/4.0/ProductionAndIndustrialFacilities.xsd schema in other datasets of the theme „Production and Industrial facilities“. But I don‘t know, how can I use this schema for priority dataset 47,01 IED/LCP emission. I think the main information in this priority dataset is emission data, but I can't find where I must put this information correctly. I think the correct way to show emissions is to fill in the amount, uom, also the name of the emission gas. In this case, The ProductionAndIndustrialFacilities GML Application Schema (version: 3 or 4) is not suitable for us to transform LCP emission data because some necessary feature types are not included in this schema (for example, ProductionInput, ProductionOutput).
For this reason, I used http://inspire.ec.europa.eu/draft-schemas/pf-core/2.0/ProductionAndIndustrialFacilitiesCore.xsd schema which suits to transform the LCP emission data, because important feature types for showing emission data (ProductionInput, ProductionOutput, ProductionProcess) are included in this schema. Unfortunately, it is not correct way (see discusion about it https://github.com/INSPIRE-MIF/helpdesk-validator/issues/519).
Perhaps you could advise what I have to do for making the GML file 100% compliant in this case?

Aurelija

**Comment by sMorrone** [comment]
Dear @AurelijaStu ,
as you correctly say, you cannot use the https://inspire.ec.europa.eu/schemas/pf/4.0/ProductionAndIndustrialFacilities.xsd to provide emission data, simply because this schema does not contain any suitable feature type. 
You can use the https://inspire.ec.europa.eu/draft-schemas/pf-core/2.0/ProductionAndIndustrialFacilitiesCore.xsd if it fits your needs. 
However, in this case, you cannot use the _Conformance class: GML application schemas, Production and Industrial Facilities_ of the INSPIRE Validator to test the conformity of your dataset to the _Data Specification on Production and Industrial Facilities_, because this test suite is meant to test data sets conformity to endorsed PF schemas - i.e. those available at https://inspire.ec.europa.eu/schemas/pf/4.0/ (that is why the error message reported by the validator states that your data set “_cannot conform to this conformance class_“). 
Nevertheless, using the INSPIRE Validator, you can still validate your dataset against those generic requirements that are not specific to a data theme/ application schema , using the test suites in the “_Interoperable data sets in GML (Guidelines for the Encoding of Spatial Data version 3.3)_” group. 
Regarding the use of https://inspire.ec.europa.eu/draft-schemas/pf-core/2.0/ProductionAndIndustrialFacilitiesCore.xsd  for the provision of priority dataset 47,01 IED/LCP, it would be important to discuss /agree this choice with the [PDS group 2016.5](https://webgate.ec.europa.eu/fpfis/wikis/display/InspireMIG/Action+2016.5%3A+Priority+list+of+datasets+for+e-Reporting) , so that a consistent mapping is made across the different Member States (enhancing data interoperability and facilitating the creation of Pan-European Datasets).
This is an important aspect to take into consideration, especially because this might result in an [INSPIRE Good Practice ](https://inspire.ec.europa.eu/portfolio/good-practice-library)related to the use of the https://inspire.ec.europa.eu/draft-schemas/pf-core/2.0/ProductionAndIndustrialFacilitiesCore.xsd (or any other schema that could be deemed suitable) for the provision of emission data. 

Hope this helps
Stefania

**Comment by AurelijaStu** [comment]
Hi Stefania,
Thanks for the reply and the helpful information.
So, in this case (if I understood correctly), I can use the scheme http://inspire.ec.europa.eu/draft-schemas/pf-core/2.0/ProductionAndIndustrialFacilitiesCore.xsd until a suitable scheme for emission data will be developed.

Aurelija
